### PR TITLE
feat: Set up realtime metrics for multiple platforms

### PR DESCRIPTION
**Pull request by loewenheim**
This adds `platform` parameters to `record_project_duration` and `is_lpq_project`, meaning that spent budget and LPQ status can now be tracked per symbolication platform.

### CODE REVIEW

File: src/sentry/api/serializers/models/project.py [hunk 1]
```diff
@@ -679,14 +679,19 @@ def get_attrs(
             deploys_by_project = self.get_deploys_by_project(item_list)
 
         for item in item_list:
+            # check if the project is in LPQ for any platform
+            symbolication_degraded = False
+            for platform in ["native", "js", "jvm"]:
```
Comment (anonrig) [inline src/sentry/api/serializers/models/project.py:684]: This for loop can be simplified by using any(x == b for x in y) syntax

Comment (loewenheim) [reply src/sentry/api/serializers/models/project.py]: Good call, it's much nicer this way.

File: src/sentry/api/serializers/models/project.py [hunk 2]
```diff
@@ -684,8 +684,10 @@ def get_attrs(
             attrs[item]["has_user_reports"] = item.id in projects_with_user_reports
             if not self._collapse(LATEST_DEPLOYS_KEY):
                 attrs[item]["deploys"] = deploys_by_project.get(item.id)
-            attrs[item]["symbolication_degraded"] = realtime_metrics.is_lpq_project(
-                project_id=item.id
+            # check if the project is in LPQ for any platform
+            attrs[item]["symbolication_degraded"] = any(
+                should_demote_symbolication(platform, project_id=item.id)
+                for platform in ["native", "js", "jvm"]
```
Comment (Swatinem) [inline src/sentry/api/serializers/models/project.py:690]: I advocated for just removing this flag (initially setting to `false` for backwards compat), so might be the time to consider that?

**Review by Swatinem — APPROVED** [review]
lgtm, though in general I would prefer to use the existing enum (if its possible due to circular imports? otherwise move it to some file that does not suffer from that problem?) for this instead of having a "stringly-typed" API.

**Review by anonrig — APPROVED** [review]

**Pull request merged**